### PR TITLE
Fixing spacing and line length in a few files #899

### PR DESCRIPTION
**Pull request by ckoerner**
Trying my hand at a few more small fixes for coding style. Feedback appreciated.

### CODE REVIEW

File: src/SPARQLStore/SparqlDBConnectionProvider.php [hunk 1]
```diff
@@ -68,19 +68,24 @@ class SparqlDBConnectionProvider implements DBConnectionProvider {
 	 * @param string|null $updateEndpoint
 	 * @param string|null $dataEndpoint
 	 */
-	public function __construct( $connectorId = null, $defaultGraph = null, $queryEndpoint = null, $updateEndpoint = null, $dataEndpoint = null ) {
-		$this->connectorId = $connectorId;
-
-		if ( $this->connectorId === null ) {
-			$this->connectorId = $GLOBALS['smwgSparqlDatabaseConnector'];
+	public function __construct(
```
Comment (mwjames) [inline src/SPARQLStore/SparqlDBConnectionProvider.php]: I probably won't agree with the constructor change since this is not really readable.

Comment (mwjames) [inline src/SPARQLStore/SparqlDBConnectionProvider.php]: @JeroenDeDauw Not sure about this ruleset but I can't agree on the strict setting in terms of constructor length.

Comment (ckoerner) [inline src/SPARQLStore/SparqlDBConnectionProvider.php]: Yeah, I was kind of stumped on the right thing to do here. Should I ignore those line length warnings for now?

Comment (JeroenDeDauw) [inline src/SPARQLStore/SparqlDBConnectionProvider.php]: The line length warnings are indeed tricky. Lots of false positives.

**Comment by JeroenDeDauw** [comment]
@ckoerner so we do not want to merge the constructor change. The rest is good though. Can you modify the PR? Thanks a lot for the effort btw!

**Comment by mwjames** [comment]
I'll fix the constructor my self.

**Pull request merged**